.github/workflows/android_x64.yml

```diff
@@ -0,0 +1,203 @@
+      volumes:
+        - /mount:/mount
+    outputs:
+      images: "${{ steps.handle_docker.outputs.images }}"
+    steps:
+      - name: Checkout
+        uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
+
+      - uses: ./.github/actions/handle_docker
+        id: handle_docker
+        with:
+          images: |
+            ov_build/ubuntu_22_04_android
+          registry: 'openvinogithubactions.azurecr.io'
+          dockerfiles_root_dir: '.github/dockerfiles'
+          changed_components: ${{ needs.smart_ci.outputs.changed_components }}
+
+  Build:
+    needs: [Smart_CI, Docker]
+    timeout-minutes: 150
+    defaults:
+      run:
+        shell: bash
+    runs-on: aks-linux-16-cores-32gb
+    container:
+      image: ${{ fromJSON(needs.docker.outputs.images).ov_build.ubuntu_22_04_android }}
+      volumes:
+        - /mount:/mount
+      options: -e SCCACHE_AZURE_BLOB_CONTAINER -e SCCACHE_AZURE_CONNECTION_STRING
+    env:
+      DEBIAN_FRONTEND: noninteractive # to prevent apt-get from waiting user input
+      CMAKE_GENERATOR: 'Ninja'
+      CMAKE_CXX_COMPILER_LAUNCHER: sccache
+      CMAKE_C_COMPILER_LAUNCHER: sccache
+      SCCACHE_IGNORE_SERVER_IO_ERROR: 1
+      SCCACHE_SERVER_PORT: 35555
+      SCCACHE_ERROR_LOG: /__w/openvino/sccache_log.txt
+      SCCACHE_LOG: warn
+      OPENVINO_REPO: '/__w/openvino/openvino/openvino'
+      VCPKG_ROOT: '/__w/openvino/openvino/vcpkg'
+      BUILD_DIR: '/__w/openvino/openvino/build'
+      ANDROID_TOOLS: '/deps/android_tools'
+      ANDROID_NDK_HOME: '/deps/android_tools/ndk-bundle'
+      ANDROID_SDK_VERSION: 29
+      ANDROID_ABI_CONFIG: x86_64
+      VCPKG_TARGET_TRIPLET: x86-android
+      VCPKG_DEFAULT_BINARY_CACHE: '/mount/caches/ccache/android_x64/vcpkg_cache'
+      VCPKG_FORCE_SYSTEM_BINARIES: '1'
+      SCCACHE_AZURE_KEY_PREFIX: android_x64
+    if: "!needs.smart_ci.outputs.skip_workflow"
+    steps:
+      - name: Clone OpenVINO
+        uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
+        with:
+          path: 'openvino'
+
+      - name: Get VCPKG version and put it into GitHub ENV
+        run: echo VCPKG_VERSION=$(cat ${OPENVINO_REPO}/vcpkg.json | python3 -c "import sys, json; print(json.load(sys.stdin)['builtin-baseline'])") >> $GITHUB_ENV
+
+      - name: Init submodules for non vcpkg dependencies
+        run: |
+          pushd ${OPENVINO_REPO}
+            git submodule update --init -- ${OPENVINO_REPO}/src/plugins
+            git submodule update --init -- ${OPENVINO_REPO}/thirdparty/zlib
+            git submodule update --init -- ${OPENVINO_REPO}/thirdparty/json
+            git submodule update --init -- ${OPENVINO_REPO}/thirdparty/gtest
+            git submodule update --init -- ${OPENVINO_REPO}/thirdparty/gflags
+            git submodule update --init -- ${OPENVINO_REPO}/thirdparty/xbyak
+            git submodule update --init -- ${OPENVINO_REPO}/thirdparty/protobuf
```

Comment:
I hope that list of extra fetched submodules should be the same as for arm64 Android.